tools/intersmash-tools-provisioners/src/main/java/org/jboss/intersmash/tools/provision/openshift/PostgreSQLImageOpenShiftProvisioner.java

```diff
@@ -60,13 +70,36 @@ protected void configureContainer(ContainerBuilder containerBuilder) {
 	@Override
 	public Map<String, String> getImageVariables() {
 		Map<String, String> vars = super.getImageVariables();
+		vars.remove(POSTGRESQL_USER);
+		vars.remove(POSTGRESQL_PASSWORD);
+		vars.remove(POSTGRESQL_ADMIN_PASSWORD);
+		vars.remove("POSTGRESQL_USERNAME");
 		vars.put("POSTGRESQL_MAX_CONNECTIONS", "100");
 		vars.put("POSTGRESQL_SHARED_BUFFERS", "16MB");
 		vars.put("POSTGRESQL_MAX_PREPARED_TRANSACTIONS", "90");
+		vars.put("POSTGRESQL_DATABASE", dbApplication.getDbName());
 		// Temporary workaround for https://github.com/sclorg/postgresql-container/issues/297
 		// Increase the "set_passwords.sh" timeout from the default 60s to 300s to give the
 		// PostgreSQL server chance properly to start under high OCP cluster load
 		vars.put("PGCTLTIMEOUT", "300");
 		return vars;
 	}
+
+	@Override
+	public void customizeApplicationBuilder(ApplicationBuilder appBuilder) {
```

Comment:
Similarly to above, I'd s/`customizeApplicationBuilder`/customizeApplication`here, so just the argument, is bound to the XTF inner details, rather than the name.